bundles/net.tourbook.export/src/net/tourbook/export/TourExporter.java

```diff
@@ -810,8 +810,8 @@ public GarminTrack doExport_60_TrackPoints(final ZonedDateTime trackDateTime, fi
          }
 
          if (isSpeed) {
-            final double speedValueMetersPerSecond = speedSerie[serieIndex] / 3.6; // speed km/h -> m/s
-            final double speedValue = Math.round(speedValueMetersPerSecond * 10.0) / 10.0;
+
+            final double speedValue = Math.round(UI.convertSpeed_KmhToMs(speedSerie[serieIndex]) * 10.0) / 10.0;
```

Comment:
See my other `UI.convertSpeed_KmhToMs` comment, `Math.round` is also a method which I try to avoid in large loops and do sometimes an inline

![image](https://github.com/mytourbook/mytourbook/assets/1283445/c73e1e02-99e9-4ce0-baa5-e699f823ead7)

Reply:
Same comment as above.... Is it still a valid concern with Java 11 ?